parser/main.py

```diff
@@ -62,6 +61,10 @@
 INFLUX_DEBUG_BUCKET = ENV_CONFIG["INFLUX_DEBUG_BUCKET"]
 INFLUX_PROD_BUCKET = ENV_CONFIG["INFLUX_PROD_BUCKET"]
```

Comment:
There is no "PROD" bucket anymore.

Reply:
Thats True. I have removed these lines **for now**. I removed both of them since they were not needed for my implementation anywhere. I will likely have to remove the environment variables for these too. Something to keep in mind is that if I end up fixing the problem with the not-working environment variables in a comment above **OR** I add the alternative idea in the comment directly above then I will have to add some form of these back.